### ISSUE TITLE
gulp build --watch stopped watching changes.

### ISSUE THREAD

**Issue by oskarwrobel** [ckeditor/ckeditor5#181]
Just after updating npm dependencies I've spotted that gulp doesn't refresh build after source file change. `gulp-watch` was updated yesterday https://github.com/floatdrop/gulp-watch/releases to `4.3.6`. I've tried to downgrade to `4.3.5` and it helps.

Could someone confirm it?

**Comment by Reinmar** [comment]
Checking...

**Comment by Reinmar** [comment]
Unfortunately I can confirm this.

**Comment by Reinmar** [comment]
I fixed the version temporarily: a38ce97.

**Comment by szymonkups** [comment]
Ok, I've found the root of the issue and reported it here: https://github.com/floatdrop/gulp-watch/issues/229

I found a way to make it work with current version of gulp-watch by creating globs with relative paths. We need to change [this glob definition](https://github.com/ckeditor/ckeditor5/blob/99ff2ea7ea126ec3e2845c701ece3fa5463d2c3d/dev/tasks/build/tasks.js#L101) to this:
``` javascript
const glob = path.join( path.relative( config.ROOT_DIR, dirPath ), '@(src|tests)', '**', '*' );
```
But for now - let's wait for the gulp-watch maintainer to answer the issue.

**Comment by UltCombo** [comment]
Hey, gulp-watch maintainer here. We have just published `gulp-watch@4.3.7` which fixes this issue. Please try it out and let us know if you experience any issues.

Special thanks to @szymonkups for helping us to identify the issue and test the patches.

**Comment by Reinmar** [comment]
Thanks for help, @UltCombo! :)
